frontend/src/components/AlertModal.tsx

```diff
@@ -8,6 +8,20 @@ export const AlertModal: FunctionComponent<{
   content: string | null;
   onHide: () => void;
 }> = ({ show, title, content, onHide }) => {
+  useEffect(() => {
+    if (!show) return;
+    const handleKeyDown = (e: KeyboardEvent) => {
+      if (e.key === "Enter") {
+        e.preventDefault();
+        onHide();
+      }
+    };
+    window.addEventListener("keydown", handleKeyDown);
+    return () => {
+      window.removeEventListener("keydown", handleKeyDown);
+    };
+  }, [show, onHide]);
```

Comment:
When the alert modal is shown, pressing Enter closes it. Users can still click "Ok" to close it as before. Sneaking this in for my own use. It's also a common UX pattern.